### PR TITLE
[release-1.2] 🌱 Fix verify-shellcheck script and fix findings

### PR DESCRIPTION
**Pull request by oscr**
**What this PR does / why we need it**:

The verify-shellcheck script exited after grep didn't find any matches. This fix enables the script to continue.

This will hopefully allow #7749 to merge.

### CODE REVIEW

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Review by ykakarap — APPROVED** [review]
/lgtm

File: hack/verify-shellcheck.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ fi
 
 echo "Running shellcheck..."
 cd "${ROOT_PATH}" || exit
-IGNORE_FILES=$(find . -name "*.sh" | grep "third_party\|tilt_modules")
+IGNORE_FILES=$(find . -name "*.sh" | { grep "tilt_modules" || true; })
```
Comment (fabriziopandini) [inline hack/verify-shellcheck.sh:67]: @oscr thanks for taking care of this
if I'm not wrong there are no more tilt_modules anymore, so we can cleanup this part and remove one of the two -not -path in L69

**Comment by oscr** [comment]
@ykakarap @killianmuldoon @fabriziopandini @sbueringer Really strange behavior here. This pr was approved 2h ago and it looks like it actually merged (from git history) but the pr wasn't closed. I got a message that it needed to be rebased so I rebased and pushed. After I pushed it was automatically closed, lgtm label was removed and now it says it was successfully merged.

**Pull request closed without merging**